code/game/machinery/buttons.dm

```diff
@@ -234,6 +234,11 @@
 	id = INCINERATOR_ATMOS_AUXVENT
 	req_one_access = list(ACCESS_ATMOSPHERICS, ACCESS_MAINT_TUNNELS)
 
+/obj/machinery/button/door/turbine_burning_chamber_vent
+	name = "turbine burning chamber vent control"
```

Comment:
why not just turbine vent control?

Reply:
well because is not the vent of the turbine, but is the vent of the burning chamber that supply the turbine, so the correct name would be "turbine's burning chamber vent control" but i didn't bother to 's it lol